### PR TITLE
pacman-database: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For: #5256

### CODE REVIEW

File: pages/linux/pacman-database.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# pacman --database
+
+> Arch Linux package manager utility.
+> More information: <https://man.archlinux.org/man/pacman.8>.
+
+- Display help:
+
+`pacman --database --help`
+
+- Mark a package as non-explicitly installed:
```
Comment (bl-ue) [inline pages/linux/pacman-database.md:10]: What do you think about 'implicitly'?  Or does that deviate too much from the official terminology?
```suggestion
- Mark a package as implicitly installed:
```

File: pages/linux/pacman-database.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Mark a package as explicitly installed:
+
+`sudo pacman --database --asexplicit {{package_name}}`
+
+- Check that all the package dependencies are installed:
+
+`pacman --database --check`
+
+- Check also the repositories to ensure all specified dependencies are available:
```
Comment (bl-ue) [inline pages/linux/pacman-database.md:22]:
```suggestion
- Check the repositories to ensure all specified dependencies are available:
```
?

File: pages/linux/pacman-database.md (hunk 1 above)
Comment (navarroaxel) [reply pages/linux/pacman-database.md:10]: maybe we can use `dependency for another package`

File: pages/linux/pacman-database.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`pacman --database --check --check`
+
+- Display only error messages:
+
+`pacman --database --quiet --check`
```
Comment (bl-ue) [inline pages/linux/pacman-database.md:28]:
```suggestion
`pacman --database --check --quiet`
```

File: pages/linux/pacman-database.md (hunk 1 above)
Comment (bl-ue) [reply pages/linux/pacman-database.md:10]: Nah&mdash;I think the original is fine.  I wasn't advocating _against yours_, I was **just** advocating _for mine._ 🙂

**Review by bl-ue — APPROVED** [review]

File: pages/linux/pacman-database.md (hunk 1 above)
Comment (sbrl) [reply pages/linux/pacman-database.md:10]: `implicitly` is the correct grammatical term to use here.

File: pages/linux/pacman-database.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# pacman --database
+
+> Arch Linux package manager utility.
```
Comment (sbrl) [inline pages/linux/pacman-database.md:3]: Could this be more descriptive of the "subcommand" being talked about here?

Comment (navarroaxel) [reply pages/linux/pacman-database.md:3]:
```suggestion
> Operate on the Arch Linux package database.
> Modify certain attributes of the installed packages.
```

Comment (bl-ue) [reply pages/linux/pacman-database.md]: Oh! I was going to suggest something like this for all of the `pacman --xxx` pages.

**Review by sbrl — APPROVED** [review]
Thanks, @navarroaxel!

**Pull request merged**